index.bs

```diff
@@ -602,13 +602,18 @@ After the step
 add the steps
 
 1. Let |attributionReportingEligibility| be "<code>[=eligibility/unset=]</code>".
+1. Let |processResponse| be null.
 1. If |attributionReportingEnabled| is true and the result of [=getting supported registrars=]
      is not [=list/is empty|empty=] and |contextOrigin| [=check if an origin is suitable|is suitable=]:
          1. Set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
+         1. Set |processResponse| to these steps given a [=response=] |response|:
+             1. Run [=process an attribution eligible response=] with |contextOrigin|,
+                 |attributionReportingEligibility|, and |response|.
 
          Issue: Allow "<code>[=eligibility/navigation-source=]</code>" when automatic beacon is specified.
 
-Issue: Set [=request=]'s [=request/Attribution Reporting eligibility=] to |attributionReportingEligibility|
+Issue: Set [=request=]'s [=request/Attribution Reporting eligibility=] to |attributionReportingEligibility| and
```

Comment:
Note that our fetch integration is specified now (https://github.com/WICG/fenced-frame/pull/106; https://wicg.github.io/fenced-frame/#send-a-beacon) so I think we can resolve this issue in this PR now.

Reply:
Thanks, updated with the fetch integration. Also added an issue to move this section to fenced frame spec.